cc_licenses/templates/includes/deed_10_body.html

```diff
@@ -0,0 +1,62 @@
+{% load i18n %}
+<div id="deed-body" class="margin-vertical-bigger padding-xxl has-text-black" >
+  {% include 'includes/snippet/icon_header_snippet.html' %}
```

Comment:
I think we need to update `license_medium_title.html`. For `CC-BY-1.0` the title still reads as being a 4.0 license.

Reply:
I don't think the approach I started in license_medium_title.html is going to scale. I'm going to make a few changes to store the English title in the license record, and then we can let Django translate that to other languages.